shuup/xtheme/plugins/products.py

```diff
@@ -241,5 +250,12 @@ def get_context_data(self, context):
         return {
             "request": request,
             "title": self.get_translated_value("title"),
-            "products": products_qs
+            "products": products_qs,
+            "data_url": reverse(
+                "shuup:xtheme-product-selections-highlight",
+                kwargs=dict(
+                    product_ids=",".join(products),
```

Comment:
what is this? what is we have a lot of products? the URL has a limit in size and this looks so wrong.
Can't you render this info in DOM using a `data-attribute` ?

Reply:
Well you still need to pass those id's to the view. I make max 10 choices for this plugin. Someone who wants more can implement something for it.